powersimdata/scenario/execute.py

```diff
@@ -204,20 +203,12 @@ def __init__(self, data_access, scenario_info, grid, ct):
 
         self.REL_TMP_DIR = self._data_access.tmp_folder(self.scenario_id)
 
-    def prepare_mpc_file(self):
-        """Creates MATPOWER case file."""
-        file_path = "/".join([self.REL_TMP_DIR, "case.mat"])
-        storage_file_path = "/".join([self.REL_TMP_DIR, "case_storage.mat"])
-
-        print("Building MPC file")
-        mpc, mpc_storage = export_case_mat(self.grid)
-
-        with self._data_access.write(file_path, save_local=False) as f:
-            savemat(f, mpc, appendmat=False)
-
-        if mpc_storage is not None:
-            with self._data_access.write(storage_file_path, save_local=False) as f:
-                savemat(f, mpc_storage, appendmat=False)
+    def prepare_grid(self):
+        """Prepare grid for simulation."""
+        print("--> Preparing grid data")
+        dest_path = "/".join([self.REL_TMP_DIR, "grid.pkl"])
```

Comment:
Not sure whether we should use `os.join` here or it is taken care of in `_data_access` already.

Reply:
The same syntax is used in `prepare_profile`. I believe it is taken care of in `DataAccess`. @jenhagg will know for sure.

Reply:
Right - we use `"/".join` here since we're dealing with a pyfilesystem path that is os independent.